### PR TITLE
Update examples to include Jest

### PR DESCRIPTION
**Pull request by Ethan-Arrowood**
Also included link to my project where I am successfully implementing jest coverage and travis-ci

### CODE REVIEW

**Comment by swenyang** [comment]
@Ethan-Arrowood thanks your script saved my life

File: README.md [hunk 1]
```diff
@@ -33,6 +33,13 @@ There are optional environment variables for other build systems as well:
 * COVERALLS_RUN_AT  (a date string for the time that the job ran.  RFC 3339 dates work.  This defaults to your
 build system's date/time if you don't set it.)
 * COVERALLS_PARALLEL (more info here: https://coveralls.zendesk.com/hc/en-us/articles/203484329)
+### [Jest](https://facebook.github.io/jest/)
+- Install [jest](https://facebook.github.io/jest/docs/en/getting-started.html)
+- Use the following to run tests and push files to coveralls: 
+```
+jest --coverage && cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js && rm -rf ./coverage
```
Comment (dijonkitchen) [inline README.md]: Might want to just add `coverage/` to the `.gitignore` instead of removing the whole directory.

Comment (Ethan-Arrowood) [reply README.md]: I'll have to review my use case for this but I don't think adding coverage to my gitignore does the same thing.

Comment (dijonkitchen) [reply README.md]: OK. I figure Jest writes it there for some caching and optimization. It also allows you to check it again at a later time without re-running all the test. Small stuff, no worries.

Comment (Yeti-or) [inline README.md]: Sorry, but why not 
```
jest --coverage --coverageReporters=text-lcov | coveralls
```
?

**Comment by maracuja-juice** [comment]
Hmm I unfortunately can't get this to work. It can't find my tests somehow.

**Comment by thevtm** [comment]
I've been using @Yeti-or version and it works perfectly.

**Comment by nickmerwin** [comment]
Thank you @Ethan-Arrowood and @Yeti-or!

**Pull request merged**